src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,196 @@
+package duke.task;
+
+import duke.*;
```

Comment:
Avoid using wildcard imports. Consider importing each class you require from the package.